miscellaneous/csv.md

```diff
@@ -0,0 +1,195 @@
+# CSV
+
+## Introduction
+
+Phlex also has a CSV table builder interface that should make you feel right at home if you're already comfortable working within Phlex. It carries over Phlex's design principles of safety, speed, and streamability.
+
+Let's take a quick look at a CSV representing an export of users.
+
+```ruby
+class UserExport < ApplicationCSV
```

Comment:
I wonder if we should use `BaseCSV` to be more consistent with how we name the base component.